app/controllers/projects_controller.rb

```diff
@@ -82,9 +82,13 @@ def create
   # PUT /projects/1.json
   def update
     @project = Project.find(params[:id])
+    existing_project_code = @project.project_code
 
     respond_with(@project) do |format|
       if @project.update_attributes(project_params)
+        if @project.project_code != existing_project_code
+          NotificationMailer::notify_team_of_project_code_alteration(@project).deliver_now
```

Comment:
`::` is name spacing in ruby, you're calling a method so it will need to be `NotificationMailer.notify_team_of_project_code_change`